Lib/gftools/packager.py

```diff
@@ -921,10 +918,8 @@ def _create_or_update_metadata_pb(upstream_conf: YAML,
     font.name = upstream_conf['name']
   metadata.designer = upstream_conf['designer']
 
-  while metadata.category:
-    metadata.category.pop()
-  for cat in upstream_conf['category']:
-    metadata.category.append(cat)
+  metadata.ClearField('category')
+  metadata.category[:] = upstream_conf['category']
```

Comment:
The `ClearField` line is unnecessary because the following line reassigns the entire list, it's not an append.

```py
metadata.category[:] = upstream_conf['category']
```

Reply:
Thanks. Updated.